src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -39,15 +43,22 @@ static void OnBackButonBehaviorPropertyChanged(BindableObject bindable, object o
 				SetInheritedBindingContext(newHandlerBehavior, bindable.BindingContext);
 		}
 
-		/// <summary>Bindable property for attached property <c>PresentationMode</c>.</summary>
+		/// <summary>
+		/// Define the small navigation animation that occurs when a page is navigated to with the GoToAsync method.
+		/// </summary>
 		public static readonly BindableProperty PresentationModeProperty = BindableProperty.CreateAttached("PresentationMode", typeof(PresentationMode), typeof(Shell), PresentationMode.Animated);
 
-		/// <summary>Bindable property for attached property <c>FlyoutBehavior</c>.</summary>
+		/// <summary>
+		/// The flyout can be accessed through the hamburger icon or by swiping from the side of the screen. 
```

Comment:
This property doesn't really relate to the hamburger icon. 

This is used to 
"lock" the flyout page so it's not a flyout.
"Hide" the flyout so there is no flyout at all
"flyout" which just means act like a normal flyout

Reply:
Added.

"Manages the behavior used to open the flyout."